docs/end-user-topics/cgc-launch-with.rst

```diff
@@ -0,0 +1,55 @@
+Cancer Genomics Cloud Launch-With
+=================================
+
+Dockstore integrates with the `Cancer Genomics Cloud <https://cgc.sbgenomics.com>` (CGC)
+platform, allowing you to launch CWL-based workflows from Dockstore in the CGC. Here is
+some information on what that looks like from a user point of view in a mini tutorial.
+
+Exporting into CGC
+------------------
+
+When browsing CWL workflows from within Dockstore, you will see a
+"Launch with CGC" button on the right. The currently selected version
+of the workflow will be exported.
+
+.. figure:: /assets/images/docs/cgc/cgc_from_dockstore.png
+   :alt: CWL workflow
+
+   CWL workflow
+
+If not logged into the CGC, you will be prompted to login. Otherwise, or
+after login, you will be presented with the following screen.
+
+.. figure:: /assets/images/docs/cgc/cgc_from_dockstore2.png
```

Comment:
App URL in the figure doesn't match the first picture at all (looks like h3agatk vs UM aligner)

Reply:
Fixed